exo-deploy/terraform/aws/alb/route53.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_route53_record" "external" {
```

Comment:
Is it possible to assign the correct security group to the proper ALB at a lower level (without creating a security group for each ALB)? The only reason we bubble up the security groups is because the services and hence ALBs are created at the root level.

Or is it not a big deal to be creating a security group for each ALB?